extension/dialog.css

```diff
@@ -97,6 +98,7 @@ body {
 
 #url-protocol.https {
     color: green;
+    color: light-dark(green, ActiveText);
```

Comment:
Do you find another [system-color](https://drafts.csswg.org/css-color/#css-system-colors) more suitable, f.e. `AccentColor` or `Highlight`, or should it be just a brighter green like [`lime`](https://developer.mozilla.org/en-US/docs/Web/CSS/named-color)?

Reply:
I think it should remain a shade of green: `https` is highlighted because it’s good (secure), and good things are conventionally green, not blue (and most definitely not red like `ActiveText`). `lime` looks good.